### PR TITLE
Restaurant Page

### PR DESCRIPTION
**Pull request by Deepakdanger**
Here I build one restaurant website using Javascript. I have Used:-

- Use of webpack to bundle JavaScript application.
- Use tabbed browsing by navbar items.
- Use of different modules.

Here I have designed 3 pages: 1) Home  2) Menu   3) Contact

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,10 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.24.0",
     "webpack": "^5.49.0",
     "webpack-cli": "^4.7.2"
   }
```
Comment (SajjadAhmad14) [inline package.json:25]: Please add the `webpack-dev-server` package as here as a dev dependency.

File: package.json [hunk 2]
```diff
@@ -10,6 +10,10 @@
   "author": "",
```
Comment (SajjadAhmad14) [inline package.json:14]: Please add all necessary run common commands like `build`, `development` should be in `package.json` inside `scripts'. 
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+## Project Screenshot
+
+![Home Page](dist/assets/img/home.PNG)
+
+![Menu Page](dist/assets/img/menu.PNG)
+
+![Contact Page](dist/assets/img/contact.PNG)
+
+
+## Live Demo Link
```
Comment (SajjadAhmad14) [inline README.md:20]: Please add an active link as this link is broken.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [ ] Git flow is used properly.
- [ ] README file has instructions to run the project locally.

## To Improve 🙏
- [ ] Please see inline comments for refference.

# Optional


### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

File: README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+
+# Restaurant
+
+This project is a small restaurant app created using javascript.Here There is 3 pages home, menu and contact.
```
Comment (rindrajosia) [inline README.md]: - [x] [OPTIONAL] Please change it to: This project is a small restaurant app created using javascript. Here There are 3 pages home, menu and contact.

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

- [ ] Please consider using tools like Grammarly to help you with your grammar. Keep in mind that your readme is the first thing users check.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**